### PR TITLE
upgrade to Selenium 3.5.3

### PR DESCRIPTION
**Pull request by niQo**
- [X] By placing an `X` in the preceding checkbox, I verify that I have signed the [Contributor License Agreement](https://github.com/SeleniumHQ/docker-selenium/blob/master/CONTRIBUTING.md#contributing-code-to-selenium)

### CODE REVIEW

**Comment by Elufimov** [comment]
I think it would be better to upgrade to https://selenium-release.storage.googleapis.com/3.5/selenium-server-standalone-3.5.1.jar

**Comment by niQo** [comment]
@Elufimov updated to 3.5.1

**Comment by Elufimov** [comment]
It was quick! I hope merge will be soon.

**Comment by Drimix20** [comment]
Version of gecko driver for firefox browser is not needed to be updated? Currently is used 0.17.0 version (see ARG GECKODRIVER_VERSION=0.17.0 in NodeFirefox/Dockerfile) but gecko driver has already released v0.18.0.

**Comment by niQo** [comment]
@Drimix20 geckodriver v0.18.0 have already been updated, see https://github.com/SeleniumHQ/docker-selenium/commit/3d29e0acd30050feadc4a937814d71b78e0e73ab#diff-79a170fb54c3005e51ce1510022006be

**Comment by Drimix20** [comment]
@niQo Can you look at master branch into NodeFirefox/Dockerfile (not Dockerfile.txt) [link](https://github.com/SeleniumHQ/docker-selenium/blob/master/NodeFirefox/Dockerfile) there is really configured variable
#============
# GeckoDriver
#============
ARG GECKODRIVER_VERSION=0.17.0

**Comment by niQo** [comment]
@Drimix20 Dockerfiles are generated from Dockerfile.txt 
see generate.sh

**Comment by barancev** [comment]
FYI: version 3.5.3 is available.

**Comment by ddavison** [comment]
hey @niQo , thanks for this!

Three favors:

1. Would you squash your commits so i can cherry pick them?
2. Would you rename `argon` to `astatine`?  (we just used argon in 3.5.0)
3. Would you perhaps update your user email and username in your local git configuration?  your commits are showing up as an unknown user.

screen shot 2017-08-30 at 2 59 12 pm

File: Hub/Dockerfile [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # NOTE: DO *NOT* EDIT THIS FILE.  IT IS GENERATED.
 # PLEASE UPDATE Dockerfile.txt INSTEAD OF THIS FILE
```
Comment (westy92) [inline Hub/Dockerfile]: Are you supposed to modify Dockerfile.txt instead?

**Comment by niQo** [comment]
@ddavison I've added username/email, changed to astatine and  squashed 3.5.3 in this commit https://github.com/SeleniumHQ/docker-selenium/pull/538/commits/c52d14a7e162424ff566bee97ccfe1fa0b47e995

**Comment by jsa34** [comment]
Any news on update to release 3.5.3?

**Comment by ddavison** [comment]
fixed in b955fcc  will close issue when released to docker hub

**Pull request closed without merging**